scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -2003,11 +2028,17 @@ class Router(formatOps: FormatOps) {
 
   private def splitWithChain(
       isFirstWith: Boolean,
-      chain: => Either[Template, Seq[Type.With]],
-      lastToken: => Token
+      owners: Set[Tree],
```

Comment:
```suggestion
      owners: => Set[Tree],
```
Let's use byName also, not sure if that wasn't an optimisation thing. We will not need to calculate it in `else`